source/db/schema.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: UTF-8
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+require_relative '../app/helpers/urls_helper'
+
+ActiveRecord::Schema.define(version: 0) do
+
+  create_table "urls", force: true do |t|
+    t.string  "linkid",      limit: UrlsHelper::LINKID_LENGTH, null: false
```

Comment:
I normally don't use code generators/wizards until I understand them. I spent a fair amount of time working to understand this and then did not go back to the migrations.

I will correct that in a new pull request.